### ISSUE TITLE
The runc can't train data

### ISSUE THREAD

**Issue by zhangchunsheng** [IndustryEssentials/ymir#637]
The runc can't train data, and there is no error in console.

<img width="1119" alt="image" src="https://user-images.githubusercontent.com/831839/168048300-3dde2e51-412a-4718-97dc-e880becc61b8.png">

**Comment by phoenix-xhuang** [comment]
can you provide more infos? e.g. how many images in your train/val dataset, is there any overlap? What's the system log? (ymir-workplace/ymir-data/logs). Also, please fill the following template

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. 
2. 
3. 
4. 

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment (please complete the following information):**
 - Server OS: [e.g. Ubuntu]
 - Ymir Version [e.g. 0.5.0]
 - Local Browser [e.g. chrome]

**Additional context**
Add any other context about the problem here.

**Comment by Zhang-SJ930104** [comment]
Which data set do you use as the training set and test set？
Our training task will detect that the training set and the test set cannot contain the same image resources.
You can first check whether there are duplicates in the image resources of the dataset.
